src/index.js

```diff
@@ -17,8 +17,9 @@ function capitalizeFirstLetter(string) {
 function checkEndpoint(url, endpoints) {
 	const parser = window.document.createElement('a');
 	parser.href = url;
+	const host = parser.host ? parser.host : window.location.host;
```

Comment:
This is not a fix.
The url given does not equal to window.location.host, it might but it is not guaranty.